Add howto docs for Cassandra Operators (issues#8189) (#9877)
mistercrunch/Airflow: docs/howto/operator/apache/cassandra.rst

 .. Licensed to the Apache Software Foundation (ASF) under one
or more contributor license agreements.  See the NOTICE file
distributed with this work for additional information
regarding copyright ownership.  The ASF licenses this file
to you under the Apache License, Version 2.0 (the
"License"); you may not use this file except in compliance
with the License.  You may obtain a copy of the License at

 ..   http://www.apache.org/licenses/LICENSE-2.0

 .. Unless required by applicable law or agreed to in writing,
software distributed under the License is distributed on an
"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
KIND, either express or implied.  See the License for the
specific language governing permissions and limitations
under the License.

Apache Cassandra Operators
======================

> 

Reviewer: Can you add a short introduction describing the service/product?
Here is an example: https://airflow.readthedocs.io/en/latest/howto/operator/google/cloud/natural_language.html

.. contents::
  :depth: 1
  :local:

Prerequisite
------------

> To use ``CassandraTableSensor`` and ``CassandraRecordSensor``, you must configure a :doc:`Cassandra Connection <../../connection/cassandra>`.

Reviewer: ```suggestion
To use operators, you must configure a :doc:`Cassandra Connection <../../connection/cassandra>`.
```
We try to avoid the listings in the introduction, because we often forget to update them. However, they are of little value here.

:ref:`howto/operator:CassandraTableSensor`

> CassandraTableSensor

Reviewer: Can you use titles that are more beginner-friendly?
Here is an example: https://airflow.readthedocs.io/en/latest/howto/operator/google/cloud/natural_language.html

Author: @mik-laj  Are these titles fine? - 
"Waiting for a Table to be created" for CassandraTableSensor
"Waiting for a Record to be created" for CassandraRecordSensor

Reviewer: +1

> -----------------

Reviewer: ```suggestion
-------------------
```

Checks for the existence of a table in a Cassandra cluster.

For parameter definition take a look at :class:`~airflow.providers.apache.cassandra.sensors.table.CassandraTableSensor`.

Use the ``table`` parameter to poke until the provided table is found. Use dot notation to target a specific keyspace.

.. exampleinclude:: /../airflow/providers/apache/cassandra/example_dags/example_cassandra_dag.py
    :language: python
    :dedent: 4
    :start-after: [START howto_operator_cassandra_table_sensor]
    :end-before: [END howto_operator_cassandra_table_sensor]

:ref:`howto/operator:CassandraRecordSensor`

CassandraRecordSensor

> -----------------

Reviewer: ```suggestion
---------------------
```
